components/service/experiments/src/main/java/mozilla/components/service/experiments/Experiments.kt

```diff
@@ -192,10 +192,11 @@ open class ExperimentsInternalAPI internal constructor() {
         assert(activeExperiment == null) { "Should not have an active experiment" }
 
         val activeExperiment = ActiveExperiment.load(context, experiments)
-        logger.info(activeExperiment?.let
-        { """Loaded active experiment from cache - id="${it.experiment.id}", branch="${it.branch}"""" }
-            ?: "No active experiment in cache"
-        )
+
+        activeExperiment?.let {
```

Comment:
Do you want/need a test for this?

Reply:
Good point, that would have helped discover this deficiency earlier

Reply:
Bug filed [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1570053)